### DISCUSSION TITLE
Error Response

### DISCUSSION THREAD

**Discussion by scgrant327** [weather-gov/api#343] [Q&A]
Since the API responds to requests in JSON format, can it also provide errors in JSON format instead of an HTML message?

**Comment by StephenClouse** [comment]
This is something we are working on, but there are a number of proxy layers to deal with internally and any one of them can decide to toss out an HTML error page under certain conditions.

**Comment by scadergit** [comment]
Towards that end, if you see an html error, that implies the request didn't get to the API. Which likely means a larger, infrastructure issue.
